docs/guides/built-ins/functions/join/example.yml

```
@@ -0,0 +1,9 @@
+api-version: reviewpad.com/v3.x
+
+workflows:
+  - name: list-security
+    description: Prints a list of security team
+    if:
+      - rule: unsecure-changes
+    then:
+      - '$comment($sprintf("This pull request contains unsecure changes. Please, request 1:1 with Security team: %s", [join($team("security"))]))'
```

The idea behind all `example.yml` is to have a fully working example. This means that anyone could copy the whole example, paste it on their `reviewpad.yml` and it's working :)

This is not the case for this example. Because the rule `unsecure-changes` is not defined. Also, the new `join` built-in is missing the `$`.

WDYT?
```suggestion
api-version: reviewpad.com/v3.x

workflows:
  - name: changes-on-security
    description: Warn when changes include security annotation
    if:
      - $hasAnnotation("security")
    then:
      - '$warn($sprintf("This pull request contains security changes. Please, request a 1:1 with security team: %s", [$join($team("security"))]))'
```